lib/smart_app_launch/token_introspection_access_token_group_stu2_2.rb

```diff
@@ -7,6 +7,9 @@ class SMARTTokenIntrospectionAccessTokenGroupSTU22 < Inferno::TestGroup
 
     id :smart_token_introspection_access_token_group_stu2_2
 
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@270',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@373'
```

Comment:
remove - @270 is a client requirement, @373 should be @273 (I think), but should go elsewhere.